crates/acct-types/src/lib.rs

```diff
@@ -41,3 +41,20 @@ pub use strata_identifiers::{
 };
 pub use util::compute_codec_sha256;
 pub use varint_vec::{VARINT_MAX, VarVec};
+
+/// Represents a system account.
+#[derive(Clone, Debug)]
+pub enum SystemAccount {
+    Zero,
```

Comment:
What is the Zero account?

Reply:
The zero account is a special system account with zero address. It's currently being used to represent the OL as a whole. For example in the checkpoint ack log to indicate it is from the OL, rather than from a particular snark account.